tools/mapping/gen_mapping_files/gen_ESMF_mapping_file/create_ESMF_map.sh

```diff
@@ -314,22 +317,24 @@ case $MACH in
         fi
         MPIEXEC="mpirun -np $NCPUS"
       fi
-      module load esmf-7.1.0r-ncdfio-mpi-O
-      module load mpt/2.15f
+      #module load esmf-${esmfvers}-ncdfio-mpi-O
+      module load esmf-${esmfvers}-ncdfio-mpt-O
+      module load mpt/2.22
     fi
     # need to load module to access ncatted
     module load nco
   ;;
-## geyser, caldera, or pronghorn
+## casper
   "dav" )
+    esmfvers=8.0.0
     module purge
-    module load intel/17.0.1 esmflibs/7.1.0r
+    module load intel/19.1.1 esmflibs/${esmfvers}
     if [ "$serial" == "TRUE" ]; then
       # No MPIEXEC
       if [ -z "$MPIEXEC" ]; then
         MPIEXEC=""
       fi
-      module load esmf-7.1.0r-ncdfio-uni-O
+      module load esmf-${esmfvers}-ncdfio-uni-O
```

Comment:
so you're using 8.0.0 on casper but 8.1.0b23 on cheyenne?

Reply:
Yes. I looked for the system installed version on casper, but we use a custom installed version on cheyenne. I don't know if we can use the same version on both?